rpc/api/eth.go

```diff
@@ -166,6 +167,20 @@ func (self *ethApi) IsMining(req *shared.Request) (interface{}, error) {
 	return self.xeth.IsMining(), nil
 }
 
+func (self *ethApi) IsSyncing(req *shared.Request) (interface{}, error) {
```

Comment:
This is one confusing method. The name of the function only makes sense when it's **not** syncing, otherwise it returns statistics.

No real suggestion here other than `SyncStats` but then it won't make much sense when it's not syncing, which doesn't return stats but instead returns a boolean :smile: